test/functional/update/delta-without-rename-flag/test.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+
+load "../../swupdlib"
+
+setup() {
+  clean_state_dir
+  clean_target_dir
+  create_manifest_tar 10 MoM
+  sign_manifest_mom 10
+  create_manifest_tar 10 os-core
+  create_manifest_tar 20 MoM
+  sign_manifest_mom 20
+  create_manifest_tar 20 os-core
+  set_os_release 10
+  chown_root -R "$DIR/target-dir" "$DIR/web-dir"
+  unpack_target_from_manifest 10 MoM
+  unpack_target_from_manifest 10 os-core
+  sudo tar -C "$DIR/web-dir/20" -cf "$DIR/web-dir/20/pack-os-core-from-10.tar" delta/ staged/
+}
+
+teardown() {
+  clean_tars 10
+  clean_tars 20
+  revert_chown_root -R "$DIR/target-dir"
+  revert_chown_root -R "$DIR/web-dir"
+}
+
+@test "using delta without a rename flag in the manifest" {
+  run sudo sh -c "$SWUPD update $SWUPD_OPTS"
```

Comment:
How are you sure this isn't falling back to a fullfile download in this test?

Reply:
I don't see anywhere in the setup where you are removing the fallback, which is how that assurance is usually met.

Reply:
Version 20 has a file (/bat) with hash `c21cc55db744a24867470b7343bdc6bde1c19e4b5a782fd82c84a86b55de6d54` that is not available as a fullfile in the test, only as a delta from a file present in version 10. If the delta fails, we would not be able to have "/bat" and the update would fail (also the output would be affected).

Reply:
Gotcha